src/main/java/ch/njol/skript/tests/TestResults.java

```diff
@@ -31,13 +31,15 @@ public class TestResults {
 	 * Succeeded tests.
 	 */
 	private final Set<String> succeeded;
+	private final boolean docs_failed;
```

Comment:
Why not something like `docsFailed`?

Also goes for other occurrences of this name case.